android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/datatypes/DoorStatusTests.java

```diff
@@ -34,7 +32,7 @@ public void testRpcValues() {
 
         // Valid Tests
         assertTrue(Validator.validateGrid(msg.getLocation(), location));
-        assertEquals(TestValues.MATCH, TestValues.GENERAL_DOOR_STATUS_TYPE, status);
+        assertEquals(TestValues.MATCH, TestValues.GENERAL_GRID, location);
```

Comment:
@kostyaBoss Below line 35 you can bring back the line:

`assertEquals(TestValues.MATCH, TestValues.GENERAL_DOOR_STATUS_TYPE, status);`